etc/fibo-vue/src/views/Courses.vue

```diff
@@ -1,313 +1,111 @@
+                        </p>
+                    </div>
+                    <h3>
+<span>Offer</span>
+</h3>
+                    <div style="text-align: justify;">
+                        <p>EDMC offers courses in Knowledge Graph, available in multiple forms:</p>
+                        <ol>
+                            <li>Instructor led virtual training course by one of the EDM Council&rsquo;s certified trainers. These courses are open enrollment, registration at the EDM Council web site,</li>
+                            <li>Enterprise focused instruction. The same course material and instructor as our open enrollment course, but with the audience limited to knowledge workers from your organization. This allows a more specific focus on use cases and industries of interest to your needs.</li>
+                            <li>Formal ELearning courses (to be available summer, 2020). The EDM Council is developing a Certification Package in Knowledge Graph. This is a deep-dive into the full details of knowledge graph technology and business practice.</li>
+                        </ol>
+                    <h4>
+<span>Open Knowledge Graph and FIBO Fundamentals Course</span>
+</h4>                        
+                        <p>In these challenging times EDMC will provide on <b>April 28-20</b> and <b>May 20-21</b> with&nbsp;<b><em>Open Knowledge Graph and FIBO Fundamentals Course</em></b>.</p>
+                        <h5><span>Open Knowledge Graph and FIBO Fundamentals Course</span></h5>
+                        <p>This course will be virtually delivered on two consecutive days, 9:00am &ndash; 3:00pm Eastern Time each day&nbsp;with breaks.</p>
+                        <p>It will consist of 5 main modules:</p>
+                        <ul>
+                            <li><b>A. Knowledge Graph Business Case and Market Adoption</b> [2:15]
+                                <ul>
+                                    <li><b>A1.</b> Knowledge Graph Perspective On Data</li>
+                                    <li><b>A2.</b> Knowledge Graph Industry Use Cases</li>
+                                    <li><b>A3.</b> Knowledge Graphs Takeaways</li>
+                                </ul>
+                            </li>
+                            <li><b>B. Knowledge Graph Technical Framework</b> [3:00]
+                                <ul>
+                                    <li><b>B1.</b> Knowledge Graph Enterprise Architecture Overview</li>
+                                    <li><b>B2.</b> <b>B3.</b> & <b>B6.</b> Knowledge Graph Data Languages (RDF, RDFS, SKOS)</li>

```

Comment:
Not sure why B6. is out of sequence. What happened to B7?
"&" on is own could cause a problem with some browsers since it represents the escape character introducing an XML entity.

Reply:
There's a full map at the bottom - 7 is SHACL, which is not included in this training.  B6 is out of order because SKOS is on a parallel track to OWL, but we are including them both.  The full structure is given in the racetracks at the bottom of the page.